src/test/features/ccj/routes/pay-by-set-date.ts

```diff
@@ -143,33 +137,4 @@ describe('CCJ - Pay by set date', () => {
       })
     })
   })
-
-  describe('validation', () => {
```

Comment:
@rattur, please do not add validation unit tests into files dedicated to route tests, that is not the right place.